packages/commons/src/interop-token/utils.ts

```diff
@@ -16,3 +16,9 @@ const bufferB64UrlEncode = (b: Buffer): string =>
     .replace(/=/g, "")
     .replace(/\+/g, "-")
     .replace(/\//g, "_");
+
+/**
+ * Decode a url encoded base64 string (RFC 7515 Appendix C) to a Uint8Array
```

Comment:
Comment mismatch with the actual function behavior. Also, it's used only in the tests, so I would move this into the commons-test testUtils

```suggestion
 * Decode a url encoded base64 JSON string (RFC 7515 Appendix C) to a JavaScript object
```

Reply:
This file contains utility functions. In my opinion, these functions shouldn’t be tied to a specific scope and could be treated as library functions. Additionally, in this particular case, it makes more sense to keep the encoding and decoding functions for the same object in the same file. This helps avoid potential future duplication, especially if someone is unaware of a function declared in another file. If it's not mandatory I prefer to leave it in this file.